src/engine/platform/mixer.h

```diff
@@ -137,6 +137,26 @@ Channel *fetch_channel(u32 channel_id);
 // Activates delay on the channel with the specified settings.
 void Channel::set_delay(f32 feedback, f32 len_seconds);
 
+///*
+// De-activates delay on the channel.
+void Channel::remove_delay();
+
+///*
+// Activates a lowpass filter on the channel with the specified weight. A
+// lowpass filter only lets sounds with a certain frequency or lower through
+// and essentialy filters the high end. This filter is about as basic as it
+// gets and is implemented with a running sum of the previous values. The
+// weight specifies how much the sum is weighted towards the new value,
+// so a lower weight means more filtering. You could probably do some math
+// and figure out / visualize the cutoff for different weights, but knowing
+// that a weight of 1 filters nothing and a weight of 0 filters everything
+// is a good starting point for testing what works for your assets and needs.
+void Channel::set_lowpass(f32 weight);
```

Comment:
This is a bit wordy, If the limits on the filter are 0 and 1 maybe assert the input value is in the given range?